solidity/contracts/bridge/Bridge.sol

```diff
@@ -434,31 +438,47 @@ contract Bridge is Governable, EcdsaWalletOwner {
     ///         with the given wallet, that has timed out. The redemption
     ///         request is identified by the key built as
     ///         `keccak256(walletPubKeyHash | redeemerOutputScript)`.
-    ///         The results of calling this function: the pending redemptions
-    ///         value for the wallet will be decreased by the requested amount
-    ///         (minus treasury fee), the tokens taken from the redeemer on
-    ///         redemption request will be returned to the redeemer, the request
-    ///         will be moved from pending redemptions to timed-out redemptions.
-    ///         If the state of the wallet is `Live` or `MovingFunds`, the
-    ///         wallet operators will be slashed.
-    ///         Additionally, if the state of wallet is `Live`, the wallet will
-    ///         be closed or marked as `MovingFunds` (depending on the presence
-    ///         or absence of the wallet's main UTXO) and the wallet will no
-    ///         longer be marked as the active wallet (if it was marked as such).
+    ///         The results of calling this function:
+    ///         - the pending redemptions value for the wallet will be decreased
+    ///           by the requested amount (minus treasury fee),
+    ///         - the tokens taken from the redeemer on redemption request will
+    ///           be returned to the redeemer,
+    ///         - the request will be moved from pending redemptions to
+    ///           timed-out redemptions.
```

Comment:
Non-blocking but let's align punctuation marks:
- `,` everywhere but the last one and `.` on the last one OR
- Treat each one as a sentence, start with a capital letter and end with a dot `.`

Reply:
Done in ad7f9b2c328b3b7bbbe1da8523a66816368ab9df.